.github/workflows/mirror-main-to-master.yml

```diff
@@ -0,0 +1,17 @@
+on:
+  push:
+    branches:
+      - 'main'
+
+jobs:
+  mirror_job:
+    runs-on: ubuntu-latest
+    name: Mirror SOURCE to DEST
```

Comment:
Replace the placeholders with the actual branch names.

Reply:
Fixed, but I didn't catch this on CUB before merging, I'll open another PR.